src/drive/targets/public/index.jsx

```diff
@@ -106,7 +107,6 @@ const init = async () => {
               <Redirect from="/*" to={`files/${id}`} />
             </Router>
           )}
-          <IconSprite />
```

Comment:
why did you remove `IconSprite`? `PublicLayout` provides the icons?

Reply:
Exactly! https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/public/PublicLayout.jsx#L10